config-windows.md

```diff
@@ -3,6 +3,21 @@
 This document describes the schema for the [Windows-specific section](config.md#platform-specific-configuration) of the [container configuration](config.md).
 The Windows container specification uses APIs provided by the Windows Host Compute Service (HCS) to fulfill the spec.
 
+## <a name="configWindowsLayerFolders" />LayerFolders
+
+**`layerfolders`** (array of strings, REQUIRED) specifies a list of layer folders the container image relies on. The list is ordered from topmost layer to base layer.
```

Comment:
Just to be clear, this is something that is not covered by [`mounts`][1] or by [image-spec unpacking][2]?  These layers need to be assembled by the runtime at container-creation time?

[1]: https://github.com/opencontainers/runtime-spec/blob/4aed614c791f4fe291baa2c7cc3dde8734be0876/config.md#example-windows
[2]: https://github.com/opencontainers/image-spec/blob/v1.0.0-rc5/layer.md#applying-changesets